### ISSUE TITLE
How can I add your overlayx to my build ?

### ISSUE THREAD

**Issue by LeOS-GSI** [MichaelSebero/Vanguard-Browser#2]
I like your overlay and I'm building my own version of Librewolf. Could you please tell me how/where I have to locate your sources in librewolf sources ? 

I have tried to add the in root folder of librewolf sources, but your overlay is not available after rebuilding 

thx Harvey
leos-gsi.de

**Comment by MichaelSebero** [comment]
What is your OS? and have you looked through the **Download Vanguard** section?

**Comment by LeOS-GSI** [comment]
sorry, don't understand your answer. I'm **_BUILDING_** on linux Mint. And about what Download section are you talking ?? I want use your sources https://github.com/MichaelSebero/Vanguard-Browser and my question was, how to integrate in 'official' librewolf sources

**Comment by MichaelSebero** [comment]
https://github.com/MichaelSebero/Vanguard-Browser/releases/tag/Vanguard-Browser

**Comment by LeOS-GSI** [comment]
Sorry, it seems we are speaking 2 different languages. I don't want use your browser. I want build my own version.

Ok, I will use other,working sources
